library/Icinga/Application/ApplicationBootstrap.php

```diff
@@ -531,7 +531,7 @@ protected function loadConfig()
      */
     protected function setupErrorHandling()
     {
-        error_reporting(E_ALL | E_STRICT);
+        error_reporting((E_ALL | E_STRICT) & ~ E_DEPRECATED);
```

Comment:
Mh you should see them imho. This allows to react sooner on possibly removed things, especially if you are really testing the new "hot shit".

Reply:
Just testing. Don't panic.

Reply:
Then edit the PR's title to include "WIP: " as prefix :)